### ISSUE TITLE
[RC] Broken link in  doc/ref_cert/index.rst 

### ISSUE THREAD

**Issue by CsatariGergely** [cntt-n/CNTT#2914]
Build is failing due
/ref_cert/index.rst.rst:105:broken link: https://build.opnfv.org/ci/job/cntt-latest-zip/ (404 Client Error: Not Found for url: https://build.opnfv.org/ci/job/cntt-latest-zip/)

@collivier , @steelescotr can you please look into this? This breaks the build for all pr-s.

**Comment by collivier** [comment]
Yes I saw it - It's because of Airship archieving. I'm updating the link.

**Comment by CsatariGergely** [comment]
@collivier thanks for updating the link. 
However Airship installer is not yet archived by the TSC (@scottsteinbrueck will initiate the process. ) I think it is timely to unrefer it from RC.
